code/modules/organs/blood.dm

```diff
@@ -127,9 +127,14 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 			if(!(temp.status & ORGAN_BLEEDING) || temp.status & ORGAN_ROBOT)
 				continue
 			for(var/datum/wound/W in temp.wounds) if(W.bleeding())
-				blood_max += W.damage / 40
-			if (temp.open)
-				blood_max += 2  //Yer stomach is cut open
+				if(src.get_species() == "Vaurca")
```

Comment:
Mistake here.

```
if (temp-open)
    blood_max += 2
```

Is originally written as **out** of the for() loop. In your case, it's written **into** the for() loop. This would increase blood_max by two for every wound in the organ, which is not the intended feature.

I can't actually dicern what your intention with this is? Could you clarify, so we could make this loop actually serve a purpose?